packages/storage-plus/src/keys.rs

```diff
@@ -52,8 +55,9 @@ impl<'a> PrimaryKey<'a> for &'a str {
 // use generics for combining there - so we can use &[u8], PkOwned, or IntKey
 impl<'a, T: PrimaryKey<'a> + Prefixer<'a>, U: PrimaryKey<'a>> PrimaryKey<'a> for (T, U) {
     type Prefix = T;
+    type SubPrefix = ();
 
-    fn key<'b>(&'b self) -> Vec<&'b [u8]> {
+    fn key(&self) -> Vec<&[u8]> {
```

Comment:
And all that was unneeded?
Looks like a better clippy/linter got this working well

Reply:
This is something the idea built-in linter recommends. These are called "elided lifetimes", and the linter's recommendation is to remove them.